.gitignore

```diff
@@ -58,4 +58,5 @@ yarn-debug.log*
 yarn-error.log*
 
 client_secret.json
-.clasp.json
\ No newline at end of file
+.clasp.json
+/backend/src/main/resources/application.yml
```

Comment:
Why? It's our main configuration file

Reply:
Is it because you're storing the email configuration there?

If so, we can do it by putting it in `/resources/emails.yml` and adding that to gitignore

Reply:
No, because generally this file contains project-based configuration and it should not be public

Reply:
It's a possible source of bugs and problems, because your changes in `application.yml` won't be propagated to everyone elses local repository.

There's nothing wrong with making project-based configuration public as long as we don't store tokens/api keys there. These should be put into `resources` and gitignored, because they rarely change.